src/controllers/transfers.js

```diff
@@ -343,7 +402,7 @@ async function putFulfillment (ctx) {
 async function getFulfillment (ctx) {
   const id = ctx.params.id
   requestUtil.validateUriParameter('id', id, 'Uuid')
-  ctx.body = await model.getFulfillment(id.toLowerCase())
+  ctx.body = (await model.getFulfillment(id.toLowerCase())).condition_fulfillment
```

Comment:
this should return the fulfillment data too, so retries don't just get part of the info back

Reply:
Added a `GET /fulfillment2` endpoint that returns both.